lib/seahorn/OperationalSemantics.cc

```diff
@@ -0,0 +1,52 @@
+
+#include "seahorn/OperationalSemantics.hh"
+#include "seahorn/Support/SeaDebug.h"
+
+#include "clam/CfgBuilder.hh"
+#include "clam/Clam.hh"
+#include "clam/ClamQueryAPI.hh"
+#include "clam/SeaDsaHeapAbstraction.hh"
+
+#include "seadsa/ShadowMem.hh"
+
+namespace seahorn {
+using namespace llvm;
+
+void OperationalSemantics::mkCrabAnalysis(
+    const llvm::Module &M, seadsa::GlobalAnalysis &dsa,
+    llvm::TargetLibraryInfoWrapperPass &tli) {
+
+  std::unique_ptr<clam::HeapAbstraction> heap_abs(
+      new clam::SeaDsaHeapAbstraction(M, dsa));
+
+  // -- Set parameters for CFG
+  clam::CrabBuilderParams cfg_builder_params;
+  LOG("bmc-crab", cfg_builder_params.print_cfg = true;);
```

Comment:
log level clearly indicates that this does not belong here . OperationalSemantics has many uses outside of bmc